README.md

```diff
@@ -102,7 +102,9 @@ from the ktlint [README](https://github.com/shyiko/ktlint/blob/master/README.md)
 
 - Download ktlint:
 
-  `curl -sSLO https://github.com/shyiko/ktlint/releases/download/0.29.0/ktlint && chmod a+x ktlint`
+  `curl -sSLO https://github.com/shyiko/ktlint/releases/download/0.29.0/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/local/bin/`
+
+  On macOS (or [Linux](http://linuxbrew.sh/)) you can also use [brew](https://brew.sh/) - `brew install shyiko/ktlint/ktlint`
```

Comment:
Is this "must" ?

Reply:
I am actually not sure if it is something specific to ZSH or not, so for now I can only say that it is a "must" with my setup of using ZSH, though I am not above the idea that it may be user error. 

This is installation instruction is directly from the ktlint readme.  I was not able to run `ktlint --apply-to-idea-project --android` out of the box with the current instructions from the Sunflower Readme.